test/AutoDiff/activity_analysis.swift

```diff
@@ -88,7 +88,7 @@ func testArrayUninitializedIntrinsicGeneric<T>(_ x: T, _ y: T) -> [T] {
 // CHECK: [NONE]   // function_ref _allocateUninitializedArray<A>(_:)
 // CHECK: [ACTIVE]   %6 = apply %5<T>(%4) : $@convention(thin) <τ_0_0> (Builtin.Word) -> (@owned Array<τ_0_0>, Builtin.RawPointer)
 // CHECK: [ACTIVE] (**%7**, %8) = destructure_tuple %6 : $(Array<T>, Builtin.RawPointer)
-// CHECK: [VARIED] (%7, **%8**) = destructure_tuple %6 : $(Array<T>, Builtin.RawPointer)
+// CHECK: [ACTIVE] (%7, **%8**) = destructure_tuple %6 : $(Array<T>, Builtin.RawPointer)
```

Comment:
This now doesn't quite make sense to me: Why is `Builtin.RawPointer` marked active? The previous "set activity across array initialization" logic was working fine and it would make `%8` have `[NONE]` activity.

Reply:
Undid the change in https://github.com/apple/swift/pull/28225/commits/86a7b97c3dae302c1014eb2970f6b6f2290f9499. I found that propagating usefulness through the `RawPointer` returned by `array.uninitialized_intrinsic` applications fixes a version of TF-952, but I can explore that in a follow-up.